src/components/language-selector/language-selector.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Box from '../box/box.jsx';
+import languages from '../../languages.json';
+import languageIcon from './language-icon.svg';
+import styles from './language-selector.css';
+
+const LanguageSelector = ({
+    currentLocale,
+    onChange,
+    ...props
+}) => (
+    <Box {...props}>
+        <div className={styles.group}>
+            <img
+                className={styles.languageIcon}
+                src={languageIcon}
+            />
+            <select
+                className={styles.languageSelect}
+                value={currentLocale}
+                onChange={onChange}
+            >
+                {Object.keys(languages).map(loc => (
```

Comment:
Good catch, that was left over because originally currentLocale was just called locale and lint complained.